processor/groupbytraceprocessor/event_test.go

```diff
@@ -48,11 +48,11 @@ func TestEventCallback(t *testing.T) {
 		{
 			casename: "onTraceExpired",
 			typ:      traceExpired,
-			payload:  pdata.NewTraceID([]byte{1, 2, 3, 4}),
+			payload:  pdata.NewTraceID([16]byte{1, 2, 3, 4, 1, 2, 3, 4, 1, 2, 3, 4, 1, 2, 3, 4}),
```

Comment:
In the batchpertrace, you did this:

```
pdata.NewTraceID([16]byte{1, 2, 3, 4})
```

Can we keep it like that here as well? IMO, it's easier to visualize the differences between the cases.

Reply:
Done.